warehouse/utils/utils.go

```diff
@@ -832,7 +832,7 @@ func WriteSSLKeys(destination backendconfig.DestinationT) WriteSSLKeyError {
 		return WriteSSLKeyError{fmt.Sprintf("Error creating SSL root directory for destination %s %v", destination.ID, err), "dest_ssl_create_err"}
 	}
 	combinedString := fmt.Sprintf("%s%s%s", clientKey, clientCert, serverCert)
-	h := sha1.New()
+	h := sha512.New()
```

Comment:
The impact of this change is that certificate folders will get invalidated and written again.

Since the checksum is used to avoid repeating overriding the files all the time, this change should not cause any issue.